concordium-node/src/p2p/peers.rs

```diff
@@ -169,6 +173,80 @@ fn calculate_average_throughput(
     Ok((avg_bps_in, avg_bps_out))
 }
 
+/// The lmdb store name of the persisted peers.
+const PEERS_STORE_NAME: &str = "peers";
+
+/// A peer that is stored in the lmdb database.
+/// This is just a newtype over a [`SocketAddr`] so
+/// it is possible to implement the necessary serializing/deserializing traits
```

Comment:
I don't think this is necessary anymore since SocketAddr has the relevant instances already.

I think it makes more sense to remove it.

Reply:
Yes. Removing.